stdlib/public/Platform/tgmath_derivatives.swift.gyb

```diff
@@ -261,3 +262,26 @@ func _${derivative_kind}Erfc(_ x: ${T}) -> (value: ${T}, ${linear_map_kind}: (${
 %    end # if T == 'Float80':
 %  end # for T in ['Float', 'Double', 'Float80']:
 %end # for derivative_kind in ['jvp', 'vjp']:
+
+// Binary functions
+%for T in ['Float', 'Double', 'Float80']:
```

Comment:
I defined an extra loop for multivariate functions due to the fact that function signatures and bodies for JVP and VJP differ a bit a this reads more clean.

Integrating this definitions into the previous  loop (l. 124 ) introduced some ugly `%if` statements.

Reply:
Separate loops for unary/binary functions makes sense!